tests/benchmarks/test_server_sni.py

```diff
@@ -0,0 +1,95 @@
+import asyncio
+
+from pytest_codspeed import BenchmarkFixture
+
+from snitun.multiplexer.channel import MultiplexerChannel
+from snitun.multiplexer.core import Multiplexer
+
+from ..conftest import Client
+from ..server.const_tls import TLS_1_2
+
+
+def test_server_sending_8192_byte_messages(
```

Comment:
other than the count and message size, are these identical?
could we parameterize them?

Reply:
Wouldnt giving them a descriptive ID with `pytest.param(..` mitigate that?

Reply:
It ends up at end and then gets cut off in the codspeed UI which makes it very hard to see which benchmark is which on a laptop or mobile screen.

Reply:
I shortened the name of the test enough that it fits with the parametrize

Reply:
Thanks!